dist/cta-button/cta-button.css

```diff
@@ -90,13 +90,21 @@ span.cta-btn__cell--fixed-height svg.icon {
   overflow: visible;
   width: 1rem;
 }
+a.cta-btn--truncated,
+a.cta-btn--truncated span {
+  line-height: 1.4em;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+}
 a.cta-btn--large {
   border-radius: calc(48px / 2);
   font-size: 1rem;
   min-height: 48px;
   padding: 13px 20px;
 }
 a.cta-btn--large-truncated {
+  border-radius: calc(48px / 2);
```

Comment:
It seems like we're setting border radius on different variations. Don't all large buttons need the same border radius? If so, can we just set the `border-radius: @button-border-radius-large` on `button.btn--large` and remove them from the different variations?

Reply:
Yeah, agree that would be a clean approach.